site/en/docs/search_reranking_using_embeddings.ipynb

```diff
@@ -31,19 +31,6 @@
         "# limitations under the License."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.      Arguments:
Line #5.        search_queries: List of search queries which.
</pre><p> </p><p>The updated docstring for <code>search_queries</code> is an incomplete sentence. Could you just drop this bit, the args seem self explanatory? Or is it needed for introspection?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Self explainatory is fine.</p>